source/NVDAObjects/window/winword.py

```diff
@@ -1430,10 +1466,520 @@ def _get_WinwordWindowObject(self):
+			label=label.replace(u"²", _( " square " ))
+			label=re.sub(r'([a-zA-Z]+)([2])',r'\1 square', label)
+			label=re.sub(r'([a-zA-Z]+)([3])',r'\1 cube', label)
+			label=re.sub(r'([a-zA-Z]+)([-]*[04-9][0-9]*)',r'\1 to the power \2', label)
+			#Translators: Substitute - by minus in trendline equations.
+			label=label.replace(u"-",_(" minus "))
+			# Translators: This message gives trendline type and name for selected series
+			output=_("{seriesName} trendline type: {trendlineType}, name: {trendlineName}, label: {trendlineLabel} ").format(seriesName=self.wordChartObject.SeriesCollection(self.seriesIndex).Name, trendlineType=self._trendlineTypeMap[self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Type], trendlineName=self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Name, trendlineLabel=label)
+		else:
+			# Translators: This message gives trendline type and name for selected series
+			output=_("{seriesName} trendline type: {trendlineType}, name: {trendlineName} ").format(seriesName=self.wordChartObject.SeriesCollection(self.seriesIndex).Name, trendlineType=self._trendlineTypeMap[self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Type], trendlineName=self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Name)
+		return output
+
+	def _get_role(self):
+		return controlTypes.ROLE_CHARTELEMENT
+
+	def invokeTrendline(self, trendlineIndex):
+		obj=WordChartSeriesTrendline(windowHandle=self.windowHandle, wordApplicationObject=self.wordApplicationObject, wordChartObject=self.wordChartObject, keyIndex=self.keyIndex, seriesIndex=self.seriesIndex, trendlineIndex=trendlineIndex)
+		self.wordChartObject.SeriesCollection(self.seriesIndex).Trendlines(self.trendlineIndex).Select()
+		eventHandler.queueEvent('gainFocus',obj)
+
+	def script_previousTrendline(self, gesture):
+		if self.trendlinesCount > 1:
+			if self.trendlineIndex==1:
+				self.trendlineIndex=self.trendlinesCount
+			else:
+				self.trendlineIndex=self.trendlineIndex-1
+			self.invokeTrendline(self.trendlineIndex)
+
+	def script_nextTrendline(self, gesture):
+		if self.trendlinesCount > 1:
+			if self.trendlineIndex==self.trendlinesCount:
+				self.trendlineIndex=1
+			else:
+				self.trendlineIndex=self.trendlineIndex+1
+			self.invokeTrendline(self.trendlineIndex)
+
+	__gestures = {
+				"kb(laptop):leftArrow":"previousTrendline",
+				"kb(desktop):leftArrow":"previousTrendline",
+				"kb(laptop):rightArrow":"nextTrendline",
+				"kb(desktop):rightArrow":"nextTrendline",
+	}
+
+class WordChartElement(WordChart):
+	_axisMap={_chartConstants.xlCategory: {_chartConstants.xlPrimary: 'Primary Category Axis', _chartConstants.xlSecondary: 'Secondary Category Axis'},
+				_chartConstants.xlValue: {_chartConstants.xlPrimary: 'Primary Value Axis', _chartConstants.xlSecondary: 'Secondary Value Axis'},

```

Comment:
Added translation comments.
Changed code is in branch in_t1541_Fin
Thanks a lot for the code review.